inclusive-design/acaw-cama
# feat: use Biome

## Description

- Removes unused localization files.
- Uses experimental [`biome-config-fluid`](https://github.com/greatislander/biome-config-fluid/) and [Biome](https://biomejs.dev/) for JavaScript and JSON linting and formatting.

## Steps to test

1. Run linting: `npm run lint`

**Expected behaviour:** Linting and formatting works as expected for JavaScript and JSON files based on [these rules](https://github.com/greatislander/biome-config-fluid/blob/main/biome.jsonc).

## Additional information

YML linting/formatting is no longer in place as this was accomplished using an ESLint plugin (which always seemed a bit counterintuitive).

## Related issues

Not applicable.
